packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -15,67 +14,77 @@ import {
   DashboardButton,
 } from './elements';
 import { Explore } from './Explore';
-import {
-  CodeSandboxIcon,
-  PlusIcon,
-  StarIcon,
-  UploadIcon,
-  BackIcon,
-} from './Icons';
+import { BackIcon } from './Icons';
 import { Import } from './Import';
-import { Welcome } from './Welcome';
 
 export const COLUMN_MEDIA_THRESHOLD = 1600;
 
+interface PanelProps {
+  tab: TabStateReturn;
+  id: string;
+  children: ReactNode;
+}
+
+/**
+ * The Panel component handles the conditional rendering of the actual panel content. This is
+ * done with render props as per the Reakit docs.
+ */
+const Panel = ({ tab, id, children }: PanelProps) => {
+  return (
+    <TabContent {...tab} stopId={id}>
+      {({ hidden, ...rest }) =>
+        hidden ? null : (
+          <div hidden={hidden} {...rest}>
```

Comment:
The Reakit tab uses the hidden attribute to mark certain elements as hidden. Since we're conditionally rendering this shouldn't be necessary unless Reakit uses the `hidden` attribute under the hood to handle some other attributes.

But it doesn't look like it. I'm removing it!